src/dropdown/dropdown.ts

```diff
@@ -57,7 +57,9 @@ export class NgbDropdownItem {
     '(keydown.ArrowUp)': 'dropdown.onKeyDown($event)',
     '(keydown.ArrowDown)': 'dropdown.onKeyDown($event)',
     '(keydown.Home)': 'dropdown.onKeyDown($event)',
-    '(keydown.End)': 'dropdown.onKeyDown($event)'
+    '(keydown.End)': 'dropdown.onKeyDown($event)',
+    '(keydown.Enter)': 'dropdown.onKeyDown($event)',
+    '(keydown.Space)': 'dropdown.onKeyDown($event)'
```

Comment:
I think it's time to keep a single `(keydown)` binding and create a `switch` or `[]. includes` in a local `onKeyDown($event)` that will delegate to `dropdown.onKeyDown($event)`

WDYT ?

Reply:
@benouat, I agreed and I have a refactoring of keyboard handling in works. Would ideally just keep it separate from this PR

- you'll see a comment by @fbasso soon, but actually `NgbDropdownToggle` doesn't need `(keydown.Enter|Space)` handlers at all. I'll fix that in a bit.
- I'll split the `onKeyDown` into two functions for `NgbDropdownToggle` and `NgbDropdownMenu` I guess and it should simplify things.

P.S. also `[].includes` needs a polyfill :)